### DISCUSSION TITLE
Installing renovate for a different account

### DISCUSSION THREAD

**Discussion by gigster99** [renovatebot/renovate#31231] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I have a renovate account for repositories in my personal account.
Now I need to install renovate for a new account: https://github.com/AFDSI/
I am the owner/manager of that account.
However, the AFDSI account and repos are not in the list of repos for which I can install renovate.
How to install renovate for AFDSI when it does not appear in my list of repo option?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
An admin of the org needs to go to https://github.com/apps/renovate and click the Install or  Configure butto

**Reply by gigster99** [reply]
Thank you. Did not know about that link. Have applied renovate bot to the AFDSI repos.
